test/kubemark/resources/start-kubemark-master.sh

```diff
@@ -241,6 +241,52 @@ function prepare-log-file {
 	chown root:root $1
 }
 
+# A helper function for removing salt configuration and comments from a file.
+# This is mainly for preparing a manifest file.
+#
+# $1: Full path of the file to manipulate
+function remove-salt-config-comments {
+	# Remove salt configuration.
+	sed -i "/^[ |\t]*{[#|%]/d" $1
+	# Remove comments.
+	sed -i "/^[ |\t]*#/d" $1
+}
+
+# Prepare the abac policy file from the salt template.
+function setup-abac-authz-policy-file {
```

Comment:
Which bits are you actually needing from ABAC?  We're about to remove it from e2e here: https://github.com/kubernetes/kubernetes/pull/39092, so I'd prefer not to add it.  I'd rather work on getting the bootstrap policies working.

There are a few bindings use here, with comments in each one about how reasonable they are or aren't: https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/e2e-rbac-bindings .  We're using these to fill in gaps in default subject bindings if the subjects don't match expectations based on the cluster bootstrapping discussions.